syntax/package_test.go

```diff
@@ -26,3 +43,35 @@ func TestPackageImportFromRoot(t *testing.T) {
 // func TestPackageExternalImport(t *testing.T) {
 // 	AssertCodesEvalToSameValue(t, `3`, `//http://github.com/'arr-ai'/arrai/examples/'xml.wbnf'`)
 // }
+
+func TestPackageExternalImportModule(t *testing.T) {
+	AssertCodesEvalToSameValue(t, `3`, `//github.com/ChloePlanet/'arrai-examples'/add`)
```

Comment:
OK for this PR, but we should move these refs to something under the arr-ai org.

Reply:
Yes. I'll open an issue for it.